baseplate/experiments/__init__.py

```diff
@@ -90,6 +90,18 @@ def _get_experiment(self, name):
             self._experiment_cache[name] = experiment
         return self._experiment_cache[name]
 
+    def _get_all_experiment_names(self):
+        config = self._config_watcher.get_data()
+        exp_names = []
+        if config:
+            for name, cfg in config.items():
+                exp_names.append(name)
+
+        return exp_names
+
+    def is_valid_experiment(self, name):
+        return self._get_config(name) is not None
```

Comment:
We should probably use `self._get_experiment` for this since the config could be invalid in some way.